api/CHANGELOG.md

```diff
@@ -6,6 +6,20 @@
 
 * FIXED: Remove optional parent_context from in_span 
 
+### v1.0.0.rc2 / 2021-06-23
+
+* BREAKING CHANGE: Remove optional parent_context from in_span 
+* BREAKING CHANGE: Refactor Baggage to remove Noop* 
+* BREAKING CHANGE: Total order constraint on span.status= 
```

Comment:
More detail is probably warranted here. This changed the API for `Status` creation quite significantly (for the better, but it will still trip people up).